packages/flutter/test/material/data_table_test.dart

```diff
@@ -1895,4 +1895,52 @@ void main() {
     // Go without crashes.
 
   });
+
+  testWidgets('DataTable clip behavior', (WidgetTester tester) async {
+    const Color selectedColor = Colors.green;
+    const Color defaultColor = Colors.red;
+
+    Widget buildTable({bool selected = false, required Clip clipBehavior}) {
+      return Material(
+        child: DataTable(
+          clipBehavior: clipBehavior,
+          border: TableBorder.all(
+            borderRadius: const BorderRadius.all(Radius.circular(30)),
```

Comment:
Can you verify that this radius is set as expected on the Material?

Reply:
Sure!